### PR TITLE
Create RAvatar component

### PR DESCRIPTION
**Pull request by followyouhome**
### What was a problem?

We have **v-avatar** component + plain html version around out application.

### How this PR fixes the problem?

Created **RAvatar** component to cover both **v-avatar** and html versions.

### Check lists

- [x] Readme file updated with actual description and example
- [x] Added all ARIA attributes required for component
- [x] Added tests for both browser and SSR render and for all components properties
- [x] Added story with all knobs and actions

### CODE REVIEW

File: packages/recomponents/src/components/r-avatar/r-avatar.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Avatar
+
+Used for user profile picture or at lest user initials. Support prop and slot to display additional text like user role, registration date or current status.
```
Comment (ZengJunyong) [inline packages/recomponents/src/components/r-avatar/r-avatar.md]: typo, `at lest` should be `at least`.

**Comment by ZengJunyong** [comment]
Just my thought: I noticed it's only used 4 times in Recomm. so I think it's a special component for Recomm, is it worth to move it into Recomponents?

**Comment by ZengJunyong** [comment]
I'm still a bit confused here, below HTML is from https://github.com/Rebilly/frontend-recomm/blob/7ed99f4199c6af9d7f0e3550bf27268c26751f5c/src/modules/automations/automations.vue#L15

```html
<div class="avatar avatar-description">
    <div class="avatar-image">
        <div class="avatar-icon">
            <v-icon :icon="item.icon"></v-icon>
        </div>
    </div>
    <div class="avatar-info text-left">
        <p :class="{'text-muted': item.comingSoon}">{{item.label}}</p>
        <p class="text-muted">{{item.description}}</p>
    </div>
</div>
```

so it can be replaced by RAvatar? I don't think the generated HTML is the same.

**Comment by followyouhome** [comment]
@ZengJunyong I think yes:

- [x] Icon
- [x] Info text (via slot)

html will be little bit different, but actual look should be same

File: packages/recomponents/src/components/r-avatar/r-avatar.md (hunk 1 above)
Comment (followyouhome) [reply packages/recomponents/src/components/r-avatar/r-avatar.md]: 🤦fixed

**Comment by ZengJunyong** [comment]
I noticed the demo of nuxt doesn't work
nuxt is not working

**Comment by ZengJunyong** [comment]
1.  as `html will be little bit different, but actual look should be same`, so let's keep eyes on integration.
2.  solve the conflicts

Others are OK so I approve it.

**Pull request merged**